### DISCUSSION TITLE
求助，快速安装的onebotV11适配器在nb run之后没有被载入，也无法和llonebot连接

### DISCUSSION THREAD

**Discussion by Aer1th7** [nonebot/discussions#524] [问答]
运行如图，求解
![屏幕截图 2024-10-12 015848](https://github.com/user-attachments/assets/d1aa8ff4-2207-4c63-8ea6-af27427b71bb)

**Comment by yanyongyu** [comment]
你的pyproject.toml里是否添加了ob适配器？

**Reply by Aer1th7** [reply]
谢谢您，我去仔细检查一下

**Comment by Aer1th7** [comment]
![屏幕截图 2024-10-12 204046](https://github.com/user-attachments/assets/fa8d73cc-c79c-4501-ab6d-1d2a0dadc5fd)
应该是加了的

**Reply by yanyongyu** [reply]
那你 nb run 的目录是否正确？连你的插件都没看到加载

**Reply by Aer1th7** [reply]
![屏幕截图 2024-10-12 210651](https://github.com/user-attachments/assets/d02401e8-0c56-4cd3-9159-882947e1c976)
是在这个目录下进行nb run的，插件的话我只装了一个内置的echo插件

**Reply by Aer1th7** [reply]
我用LLOneBot-Master插件解决了这个问题，打扰您了
